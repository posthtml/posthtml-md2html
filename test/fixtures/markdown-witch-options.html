<main markdown>
# Main heading

Est est consequat anim do.
Ipsum est excepteur ipsum veniam non laborum pariatur.
Laborum non eiusmod et magna elit quis esse.

## Subheading

Excepteur sunt qui nostrud fugiat magna mollit amet officia excepteur ea excepteur.
Ea eu reprehenderit ullamco commodo ullamco aute.
Sit aliquip fugiat minim velit culpa.


| This | is | a | Table |
|------|----|---|-------|
| And  | a  | - | row   |
| And  | an | o | ther row |
  </main>