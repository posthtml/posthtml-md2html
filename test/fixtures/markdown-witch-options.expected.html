<main><h1>Main heading</h1>
<p>Est est consequat anim do.<br>Ipsum est excepteur ipsum veniam non laborum pariatur.<br>Laborum non eiusmod et magna elit quis esse.</p>
<h2>Subheading</h2>
<p>Excepteur sunt qui nostrud fugiat magna mollit amet officia excepteur ea excepteur.<br>Ea eu reprehenderit ullamco commodo ullamco aute.<br>Sit aliquip fugiat minim velit culpa.</p>
<table>
<thead>
<tr>
<th>This</th>
<th>is</th>
<th>a</th>
<th>Table</th>
</tr>
</thead>
<tbody><tr>
<td>And</td>
<td>a</td>
<td>-</td>
<td>row</td>
</tr>
<tr>
<td>And</td>
<td>an</td>
<td>o</td>
<td>ther row</td>
</tr>
</tbody></table>
</main>